ssr/Cargo.toml

```diff
@@ -87,45 +87,23 @@ gob-cloudflare = { git = "https://github.com/go-bazzinga/gob-cloudflare", rev =
 yral-metadata-client = { git = "https://github.com/go-bazzinga/yral-metadata", rev = "c394bf9af3f32d81c1ac50b966c25dafafa2545b", default-features = false }
 yral-metadata-types = { git = "https://github.com/go-bazzinga/yral-metadata", rev = "c394bf9af3f32d81c1ac50b966c25dafafa2545b", default-features = false }
 gloo-utils = { version = "0.2.0", features = ["serde"] }
-tonic = { version = "0.11.0", features = [
-    "tls",
-    "tls-webpki-roots",
-], optional = true }
-prost = { version = "0.12.4", optional = true }
 hmac = { version = "0.12.1", optional = true }
 wasm-bindgen-futures = { version = "0.4.42" }
 testcontainers = { version = "0.20.0", optional = true }
 yral-testcontainers = { git = "https://github.com/go-bazzinga/yral-testcontainers", rev = "f9d2c01c498d58fca0595a48bdc3f9400e57ec2f", optional = true }
 js-sys = "0.3.69"
 tonic-web-wasm-client = { version = "0.6" }
-
-[dependencies.tonic_2]
-package = "tonic"
-version = "0.12.0"
-optional = true
-default-features = false
-features = ["prost", "codegen"]
-
-
-[dependencies.prost_2]
-package = "prost"
-version = "0.13.0"
-optional = true
+tonic = { version = "0.12.0", default-features = false, features = ["prost", "codegen"] }
```

Comment:
Is the event streaming working fine with this ? Lets check once in the preview link.